docs/prow/presets.md

```diff
@@ -24,6 +24,7 @@ This document contains the list of all Presets available in the [`config.yaml`](
 | **preset-docker-push-repository-test-infra**              | It defines the environment variable with the location of the directory in the GCR repository for storing Docker images from the `test-infra` repository.                                  |
 | **preset-docker-push-repository-control-plane**           | It defines the environment variable with the location of the directory in the GCR repository for storing Docker images from the `control-plane` repository.                                  |
 | **preset-docker-push-repository-incubator**               | It defines the environment variable with the location of the directory in the GCR repository for storing Docker images from all repositories under the `kyma-incubator` organization.     |
+| **preset-build-busola-master**                           | It defines the environment variable with the location of the directory in the Docker repository for storing Docker images from the `busola` repository. It also sets the **BUILD_TYPE** variable to `master`. |
```

Comment:
What is a `BUILD_TYPE` exactly? Is it a master branch or something else?

Reply:
BUILD_TYPE may be set to master, release, pr. And it basically set what kind of build is it